src/shared/pages/team/editTeam/editTeam.tsx

```diff
@@ -26,13 +37,29 @@ const validationSchema = object({
         .label('new team name')
 });
 
-const EditTeam: React.FC<EditTeamProps> = ({ csrfToken, history, match }) => {
+
+const EditTeam: React.FC<EditTeamProps> = ({ csrfToken, history, match, hasRole }) => {
 
     const [pageError, addFormError, clearErrors, setErrorMessage] = useError('', VALIDATION_ERROR_TITLE);
     const [state, dispatch] = useReducer(reducer, initialState);
 
     const { params: { teamId } } = match;
 
+    const getUnitsForTypeAhead = useCallback(() => new Promise<Item[]>(resolve => getUnits()
+        .then((units) => {
+            const items: Item[] = [];
+            units.forEach(function (unit){
+                items.push({ label: unit.displayName, value: unit.type });
+            });
+            resolve(items);
```

Comment:
This might be better something like:

```
const items = units.map((el) => {
	let rObj = {};
      rObj.label = el.displayName;
      rObj.value = el.type;
  
  return rObj;
})
```

to avoid mutation 

(Note: this is untested so use as inspiration)